backend/events/models.py

```diff
@@ -50,10 +54,27 @@ class Event(models.Model):
     topics = models.ManyToManyField("content.Topic", blank=True)
 
     def __str__(self) -> str:
+        """
```

Comment:
I'll be removing all of these, @rohityadav2604, as I think the code is too simple to document 🤔 Maybe there's a way to turn this off within the settings of the numpydoc validation? Could you take a look at this in the next PR? :)